app/labor/hex_comparer.rb

```diff
@@ -1,49 +1,49 @@
 class HexComparer
+  RGB_REGEX = /^#?(?<r>..)(?<g>..)(?<b>..)$/.freeze
+  ACCENT_MODIFIERS = [1.14, 1.08, 1.06, 0.96, 0.9, 0.8, 0.7, 0.6].freeze
+
   def initialize(hexes, amount = 1)
-    @hexes = hexes
+    @hexes = hexes.sort
     @amount = amount
   end
 
-  def order
-    hexes.sort
-  end
-
   def smallest
-    order.first
+    hexes.first
   end
 
   def biggest
-    order.last
+    hexes.last
   end
 
   def brightness(amount = 1)
-    rgb = smallest.delete("#").scan(/../).map(&:hex).map { |color| color * amount }.map(&:round)
-    format("#%<r>02x%<g>02x%<b>02x", r: rgb[0], g: rgb[1], b: rgb[2])
+    rgb = hex_to_rgb_hash(smallest).transform_values do |color|
+      (color * amount).round
+    end
+    format("#%<r>02x%<g>02x%<b>02x", rgb)
   rescue StandardError
     smallest
   end
 
+  # Returns the first valid hex string it finds (# + 6 digits)
   def accent
-    if brightness(1.14**amount).size == 7
-      brightness(1.14**amount)
-    elsif brightness(1.08**amount).size == 7
-      brightness(1.08**amount)
-    elsif brightness(1.06**amount).size == 7
-      brightness(1.06**amount)
-    elsif brightness(0.96**amount).size == 7
-      brightness(0.96**amount)
-    elsif brightness(0.9**amount).size == 7
-      brightness(0.9**amount)
-    elsif brightness(0.8**amount).size == 7
-      brightness(0.8**amount)
-    elsif brightness(0.7**amount).size == 7
-      brightness(0.7**amount)
-    elsif brightness(0.6**amount).size == 7
-      brightness(0.6**amount)
+    ACCENT_MODIFIERS.each do |modifier|
```

Comment:
The previous solution was (a) verbose and (b) calculated the final brightness twice, once for comparing the length, once for returning it. I instead use `each` + `break` with a return value.